Feature/omk 11062&10608

whatgwan opened the pull request:
check avx support and added a last daemon start check

whatgwan committed:
cpu info avx update

whatgwan committed:
last daemon start time

markdueck1 commented on admin/support.pl:
```diff
@@ -330,6 +342,27 @@ sub collect_evidence
+	open(my $daemon_info_fh, '>>', "$targetdir/system_status/init/daemon_status") or die "Could not open file '$targetdir/system_status/init/daemon_status' $!";
+
+        foreach my $daemon (@daemons) {
+                my $status = `systemctl is-active $daemon 2>/dev/null`;
```
do we need to see if systemctl is available on the box?

whatgwan replied:
I have added in your suggestion

markdueck1 commented on admin/support.pl:
```diff
@@ -330,6 +342,27 @@ sub collect_evidence
+
+        foreach my $daemon (@daemons) {
+                my $status = `systemctl is-active $daemon 2>/dev/null`;
+                chomp $status;
```
this should check to see if $status is something before it starts working with it

whatgwan replied:
I have added in your suggestion

Commit:
modified:   support.pl
rectified based on marks comments. check if we have systemctl, and check if status is defined before using it

Commit:
Merge branch 'nmis9_dev' into feature/omk-11062&10608
